extensions/wikia/PhalanxII/hooks/PhalanxHooks.class.php

```diff
@@ -249,7 +249,11 @@ static public function onAfterFormatPermissionsErrorMessage( Array &$permErrors,
 	 * @param User $user
 	 * @return bool true
 	 */
-	static public function onGetBlockedStatus( User $user ) {
+	static public function onGetBlockedStatus( User $user, $shouldLogBlockInStats=false, $global=true ) {
```

Comment:
:speech_balloon: — for backwards compatibility, assume we are dealing with a global block; do nothing in case of a local block.